docs/gui/journal.rst

```diff
@@ -20,22 +20,32 @@ Supported Features
 ------------------
 
 - Cursor Control: Navigate through text using arrow keys (left, right, up, down) for precise cursor placement.
-- Fast Rewind: Use 'Shift+Left'/'Ctrl+B' and 'Shift+Right'/'Ctrl+F'to move the cursor one word back or forward
+- Fast Rewind: Use :kbd:`Shift+Left`/:kbd:`Ctrl+B` and :kbd:`Shift+Right`/:kbd:`Ctrl+F` to move the cursor one word back or forward.
```

Comment:
this will render as `Shift+Left` on one "key". You should probably write this as
```
:kbd:`Shift`:kbd:`Left`
```
etc.